### DISCUSSION TITLE
Group styles in prefixes

### DISCUSSION THREAD

**Discussion by pierredewilde** [master-co/css#75] [Ideas]
In Master CSS, [group styles](https://docs.master.co/css/syntax-tutorial#group-styles) support suffixes:

`<ul class="{block;width:fit;text:center;font:24;font:blue/.5}>li:hover@md"></ul>`

It would be great to also support prefixes with `x:(,)`:

`<ul class="{block;width:fit;text:center;f:(24,blue/.5)}>li:hover@md"></ul>`

See [variant group of Windi CSS or Unocss](https://www.npmjs.com/package/@unocss/transformer-variant-group)

**Comment by 1aron** [comment]
@pierredewilde The feature is only partially implemented, so we are not going to.
You don't recognize whether the 16 in `font:{16;bold}` is font size or weight, and many more similar cases. We've support native shorthand properties: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

It's a good feature request, thanks!
